src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -646,6 +650,11 @@ fn run(mut args: Args) -> Result<(), anyhow::Error> {
         Option<Arc<dyn CloudResourceController>>,
     ) = match args.orchestrator {
         OrchestratorKind::Kubernetes => {
+            if let Some(instance_directory) = &args.instance_directory {
+                if !instance_directory.is_absolute() {
+                    bail!("kubernetes orchestrator requires an absolute `--instance-directory`");
```

Comment:
Validation of Kubernetes orchestrator-specific parameters should happen in the Kubernetes orchestrator constructor, not here.

Reply:
Hmm, actually, I think we should not do this validation at all! I can't see why it would need to be absolute for the Kubernetes orchestrator. It needs to be the _same_ for all spawned services, sure, but a relative path presumably works just fine.